examples/sftp-server/main.go

```diff
@@ -0,0 +1,132 @@
+// An example SFTP server implementation using the golang SSH package.
+// Serves the whole filesystem visible to the user, and has a hard-coded username and password,
+// so not for real use!
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+
+	"github.com/pkg/sftp"
+	"golang.org/x/crypto/ssh"
+)
+
+// Based on example server code from golang.org/x/crypto/ssh and server_standalone
+func main() {
+
+	readOnly := false
```

Comment:
try
 var (
   readOnly bool
   debufLevelStr string
   ...
)

These will then have their zero value, which will then be overwritten by flag.Parse